src/components/ui/use-toast.ts

```
@@ -0,0 +1,9 @@
+
+// Re-export toast implementations from both hooks for consistent usage
+import { useToast as useHookToast, toast as hookToast } from "@/hooks/use-toast";
+import { toast as sonnerToast } from "sonner";
+
+// Export all toast implementations for flexibility
+export const useToast = useHookToast;
+export const toast = sonnerToast;
+export const hookToast = hookToast;
```

Biome (1.9.4)

[error] Shouldn't redeclare 'hookToast'. Consider to delete it or rename it.

'hookToast' is defined here:


(lint/suspicious/noRedeclare)